README.md

```diff
@@ -1,70 +1,69 @@
+- If you just want to check the webpage/webapp, you have it available on the live demo link.
 
-### `npm run eject`
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+#### Install locally the code of the application
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+In order to install a local version of this project and please do the following steps:
+- Install the latest version of any code editor.
+- install the latest version of GIT.
+- open your terminal
+- Go to the [repository](https://github.com/ForHemer/Bookstore)  and clone it with `git clone git@github.com:ForHemer/Bookstore.git` .
+- open the repository cloned with `cd Bookstore`
+- if your code editor is VsCode type `code .`
+- Install npm dependencies with `npm install`
+- To run it type `npm start`
+- and enjoy!
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+## Author
 
-## Learn More
+👤 **Hemerson Foreste**
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+- GitHub: [@ForHemer](https://github.com/ForHemer)
+- Twitter: [@HemersonForeste](https://twitter.com/HemersonForeste)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/hemerson-foreste-890685197)
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+## 🤝 Contributing
 
-### Code Splitting
+Contributions, issues, and feature requests are welcome!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+Feel free to check the [issues page](https://github.com/ForHemer/Bookstore/issues).
 
-### Analyzing the Bundle Size
+## Show your support
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+Give a ⭐️ if you like this project!
 
-### Making a Progressive Web App
+## 📝 License
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+This project is [MIT](https://github.com/microverseinc/readme-template/blob/master/MIT.md) licensed
```

Comment:
**Nice update readme according to TSE review.**
- [ ] You should add a license in your repository and fill in your name in the license.

Reply:
Hi @ahmetbozaci, Thank you to take your time for reviewing my project and helping me to write better code, I appreciate it. But it would be nice if you put this comment as **optional**, Cause : 

- A license inside a repository with my name it's not mandatory for the approval of a project. 
- This is the beginning of the project, this suggestion could be in the next branch.

I let you imagine my frustration.

My deep respects!

Reply:
> **Nice update readme according to TSE review.**
> 
>     * [ ]  You should add a license in your repository and fill in your name in the license.



Hi @ahmetbozaci, Thank you to take your time for reviewing my project and helping me to write better code, I appreciate it. But it would be nice if you put this comment as optional, Cause :

    A license inside a repository with my name it's not mandatory for the approval of a project.
    This is the beginning of the project, this suggestion could be in the next branch.

I let you imagine my frustration.

My deep respects!

Reply:
@ForHemer  Hello friend, You are right, it should be optional. Sorry about that.